docs/configuration.asciidoc

```diff
@@ -290,19 +290,58 @@ storage use in your Elasticsearch cluster.
 
 |============
 | Environment                            | `Config` key               | Default
-| `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` | `span_frames_min_duration` | `5`
+| `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION` | `span_frames_min_duration` | `"5ms"`
 |============
 
-Use this to disable stacktrace collection for spans with a duration shorter than or equal to the given amount of milleseconds.
+Use this to disable stacktrace collection for spans with a duration shorter
```

Comment:
This is meant as the verb _connect_?

Reply:
ah yes, I misread that one. My point is though that the config option says nothing about _stacktrace_ and the explanation says nothing about _frames_. So for us this connection is clear but it might make sense to use _stacktrace frames_  in the description.